source/how-tos/create-ias-boot-image.rst

```diff
@@ -0,0 +1,116 @@
+.. _create-ias-boot-image:
+
+Create IAS Boot Image
+----------------------
+
+This page provides intructions to create a bootable Linux kernel image in IAS format and setup boot partition in |SPN| to boot from it.
+
+
+The following step-by-step instructions is an example to boot Yocto kernel using IAS image format on |UP2|.
+
+
+**Step 1:** Download and install open source IAS tool from |IAS|
+
+.. |IAS| raw:: html
+
+   <a href="https://github.com/intel/iasimage" target="_blank">here</a>
+
+
+**Step 2:** Get kernel and associated files from target Yocto disk image.
+
+Copy vmlinuz (e.g. ``vmlinuz``), initramfs (``initrd``) and get kernel command line string (e.g. /proc/cmdline) and transfer them to your development host.
+
+Create ``cmdline.txt`` with kernel command line string. A NULL or empty file can be used if no kernel command line is required.
+
+**Step 3:** Generate ``iasimage.bin`` file
+
+Run::
+
+  iasimage -i 0x30300 -o iasimage.bin -d <RSA_private_key> <cmdline.txt> <vmlinuz> <initrd>
+
+    <RSA_private_key>: use the same key located in BootloaderCorePkg/Tools/Keys/TestSigningPrivateKey.pem
+
+Sample output messages::
+
+    Creating ias-image with 3 files
+    Detected image type is (0x30000) - Multi-file boot image
+      cmd.null.txt (0 Bytes)
+      vmlinuz (6.6 MiB)
+      initrd (7.3 MiB)
+    WARNING: No public key or signature flag in image type, adding both
+    File 1 size 0 bytes
+    File 2 size 6902016 bytes
+    File 3 size 7644383 bytes
+    Calculating Checksum... Ok
+    Signing... Ok
+    Writing... Ok
+
+
+**Step 4:** Copy ``iasimage.bin`` onto the first boot partion on |UP2|
+
+
+**Step 5:** Configure boot option entry to load ``iasimage.bin`` from the boot partition
+
+Make sure the boot option is configured to the location of ``iasimage.bin`` on the boot device. E.g., if ``iasimage.bin`` is located on the first boot FAT32 partition on USB, the boot option entry should look like the following::

```

Comment:
provide a link to the boot option format page so that people can understand the meanings.